src/main/aws/sso.ts

```
@@ -0,0 +1,147 @@
+        webPreferences: {
+          nodeIntegration: false,
+          contextIsolation: true,
+        },
+      });
+
+      // Load the verification URL
+      console.log("Loading URL:", deviceAuthResponse.verificationUriComplete);
+      await authWindow.loadURL(deviceAuthResponse.verificationUriComplete!);
+
+      // Poll for token
+      const token = await this.pollForToken(deviceAuthResponse.deviceCode!, client, deviceAuthResponse.interval || 5);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)